src/test/java/uk/gov/register/views/v2/EntryListViewTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.register.views.v2;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import io.dropwizard.jackson.Jackson;
+import org.junit.Test;
+import uk.gov.register.core.Entry;
+import uk.gov.register.core.EntryType;
+import uk.gov.register.core.HashingAlgorithm;
+import uk.gov.register.util.HashValue;
+import uk.gov.register.views.representations.CsvWriter;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class EntryListViewTest {
+    private final ObjectMapper objectMapper = Jackson.newObjectMapper();
+    private final Entry entry1 = new Entry(
+            1,
+            new HashValue(HashingAlgorithm.SHA256,"ab"),
+            Instant.ofEpochSecond(1470403440),
+            "b",
+            EntryType.user
+    );
+    private final Entry entry2 = new Entry(
+            2,
+            new HashValue(HashingAlgorithm.SHA256,"cd"),
+            Instant.ofEpochSecond(1470403441),
+            "c",
+            EntryType.user
+    );
+    private final EntryListView view = new EntryListView(ImmutableList.of(entry1, entry2));
+
+    @Test
+    public void jsonRepresentation() throws JsonProcessingException {
+        String result = objectMapper.writeValueAsString(view);
+
+        assertThat(result, equalTo("[" +
+                "{" +
```

Comment:
might be more stable and readable to make assertions on the JSON value using something like Jackson rather than a string blob? Do not feel very strongly about this though.

Reply:
...apparently in Java 12 string literals are coming which would make these kinda things much nicer: https://openjdk.java.net/jeps/326